cli/azd/cmd/infra_create.go

```diff
@@ -283,15 +289,71 @@ func reportDeploymentStatusInteractive(ctx context.Context, azCli tools.AzCli, e
 	}
 
 	succeededCount := 0
+	newlyDeployedResources := []azureutil.DeployedAzureResource{}
 
-	for _, resourceOperation := range *operations {
+	for _, resourceOperation := range operations {
 		if resourceOperation.Properties.ProvisioningState == "Succeeded" {
 			succeededCount++
+
+			if !loggedResources[resourceOperation.Properties.TargetResource.Id] {
+				resourceTypeDisplayName := ""
+
+				if resourceOperation.Properties.TargetResource.ResourceType == string(infra.AzureResourceTypeWebSite) {
+					// Web apps have different kinds of resources sharing the same resource type 'Microsoft.Web/sites', i.e. Function app vs. App service
+					// We resolve it by querying the properties of the ARM resource.
+					resourceTypeName, err := resourceManager.GetWebAppResourceTypeDisplayName(ctx, env.GetSubscriptionId(), resourceOperation.Properties.TargetResource.Id)
+
+					if err != nil {
+						// Best effort -- use static translation 'Web App' if we could not resolve the exact resource type
+						resourceTypeDisplayName = infra.GetResourceTypeDisplayName(infra.AzureResourceType(resourceOperation.Properties.TargetResource.ResourceType))
+					} else {
+						resourceTypeDisplayName = resourceTypeName
+					}
+				} else {
+					resourceTypeName := infra.GetResourceTypeDisplayName(infra.AzureResourceType(resourceOperation.Properties.TargetResource.ResourceType))
+
+					if resourceTypeName != "" {
+						resourceTypeDisplayName = resourceTypeName
+					}
+				}
+
+				newlyDeployedResources = append(newlyDeployedResources, azureutil.DeployedAzureResource{
+					Id:                      resourceOperation.Properties.TargetResource.Id,
+					ResourceName:            resourceOperation.Properties.TargetResource.ResourceName,
+					ResourceType:            resourceOperation.Properties.TargetResource.ResourceType,
+					ResourceTypeDisplayName: resourceTypeDisplayName,
+					DeployedTimestamp:       resourceOperation.Properties.Timestamp,
+				})
+			}
+		}
+	}
+
+	sort.Sort(azureutil.DeployedAzureResourceByTimestamp(newlyDeployedResources))
+
+	for _, newResource := range newlyDeployedResources {
+		// Don't log resource types that are not known to us.
+		// There are two cases:
+		// 1. Resource types that are sub-components. i.e., application settings: "Microsoft.Web/sites/config"
+		// 2. Azure resources that we do not have a translation of the resource type for.
+		if newResource.ResourceTypeDisplayName != "" {
+			spinner.Println(fmt.Sprintf(
+				"%s - Created %s: %s",
+				newResource.DeployedTimestamp.Local().Format("2006-01-02 15:04:05"),
```

Comment:
I don't think we need date/time in the progress.  To me, it muddies the message and isn't necessary to communicate progress.  @savannahostrowski to comment on that.

Reply:
Customers asked for more progress information. One, for instance, said he'd go get coffee instead of wait. The timestamp shows elapsed time; things are still happening and clues customers in on "ah, the template created x,y,z that took a,b,c min each. That explains why it took so long." That said, many followed up with "debug mode" or "log file" when they said they needed more info. I am ok if we can at least get timestamp in debug mode or log file. 

Also, we showed the mockup to 2 customers last week. 50/50 - one love the information (no need to to go to the portal to get the details); one said it's too much.

Reply:
Okay, based on all the feedback thus far from multiple devs (Matt, Jon) and PM (Savannah), it sounds like we should start simple and not include the timestamp. The log files will include the timestamp.

> "ah, the template created x,y,z that took a,b,c min each"

Should we just display the duration for each resource creation? i.e: `Created App Service - app-123456 (40 s)`. This was collective feedback from Matt and Viktor. That would answer the question. I don't think it is too confusing what the duration means -- the resource creation time. We could also word it to add hints:  `Created App Service - app-123456 in 40 seconds`